README.md

```diff
@@ -65,6 +65,19 @@ If you want to set host to a specific IP address for example `172.17.12.1` run `
 - CSS developers should ensure that animation is well documented and communicated to the respective React developer.
 - Once the component/enhancement is complete it should receive sign off from a visual designer who can then update the master sketch file with any changes.
 
+## Experimental components
+
+When creating a brand new component, it should be released as experimental in order to get feedback.  This results in the css not being bundled into the full distribution (patternfly.css), but the component level css is still included in the npm package for usage.
```

Comment:
If you look on the upgrade guide or contribution pages, we usually wrap file names in code backticks. 

```suggestion
When creating a brand new component, it should be released as experimental in order to get feedback.  This results in the css not being bundled into the full distribution (`patternfly.css`), but the component level css is still included in the npm package for usage.
```